packages/date/src/test/index.js

```diff
@@ -1042,3 +1046,55 @@ describe( 'getStoreTimeZoneMoment', () => {
 		expect( utcOffset ).toHaveBeenCalledWith( '-04:00' );
 	} );
 } );
+
+describe( 'getDateFormatsForIntervalPhp', () => {
```

Comment:
I think the tests to check if both formatter producing the equivalent date format is neat!